src/runner/powertoys_events.cpp

```diff
@@ -41,6 +42,42 @@ void PowertoysEvents::unregister_receiver(PowertoyModuleIface* module) {
   }
 }
 
+void PowertoysEvents::register_system_menu_action(PowertoyModuleIface* module)
+{

```

Comment:
Use the existing coding style of this file, curly braces at the end of the line.